examples/librispeech/squeezeformer/local

```diff
@@ -0,0 +1 @@
+../s0/local
```

Comment:
`squeezeformer` in examples/librispeech is not required here, we can just do it in s0 by configue since it shares same training and decoding recipe.

Reply:
Okay, what about remove it after the whole `README.md` of squeezeformer part is updated?

Reply:
ok!